### ISSUE TITLE
TypeScript Errors

### ISSUE THREAD

**Issue by borro** [edvinv/named-js-regexp#5]
I have some error, when i build my project with this package.
TypeScript Version is 3.2.2
```
node_modules/named-js-regexp/lib/named-js-regexp.d.ts:11:3 - error TS7010: 'replace', which lacks return-type annotation, implicitly has an 'any' return type.

11  	replace(text: string, replacement: string | ((this: NamedRegExpExecArray | null, match?, ...params) => string));
    	~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/named-js-regexp/lib/named-js-regexp.d.ts:11:84 - error TS7006: Parameter 'match' implicitly has an 'any' type.

11  	replace(text: string, replacement: string | ((this: NamedRegExpExecArray | null, match?, ...params) => string));
    	                                                                                 ~~~~~~

node_modules/named-js-regexp/lib/named-js-regexp.d.ts:11:92 - error TS7019: Rest parameter 'params' implicitly has an 'any[]' type.

11  	replace(text: string, replacement: string | ((this: NamedRegExpExecArray | null, match?, ...params) => string));
    	                                                                                         ~~~~~~~~~


Found 3 errors.
```

Pull Request #6

**Comment by edvinv** [comment]
Just published version 1.3.4 should give no errors when compiling with compiler option 'noImplicitAny'.

**Comment by borro** [comment]
Good. version 1.3.4 compiling with ```"strict": true``` too.
